dash/dependencies.py

```diff
@@ -134,13 +137,31 @@ def __init__(
 class Input(DashDependency):  # pylint: disable=too-few-public-methods
     """Input of callback: trigger an update when it is updated."""
 
-    allowed_wildcards = (MATCH, ALL, ALLSMALLER)
+    def __init__(
+        self,
+        component_id: ComponentIdType,
+        component_property: str,
+        allow_optional: bool = False,
+    ):
+        super().__init__(component_id, component_property)
+        self.allow_optional = allow_optional
+        self.allowed_wildcards = (MATCH, ALL, ALLSMALLER)
 
 
 class State(DashDependency):  # pylint: disable=too-few-public-methods
     """Use the value of a State in a callback but don't trigger updates."""
 
-    allowed_wildcards = (MATCH, ALL, ALLSMALLER)
+    def __init__(
+        self,
+        component_id: ComponentIdType,
+        component_property: str,
+        allow_optional: bool = False,
+    ):
+        super().__init__(component_id, component_property)
+        self.allow_optional = allow_optional
+        self.allowed_wildcards = (MATCH, ALL, ALLSMALLER)
```

Comment:
That was a class attribute before, it doesn't change on the object and I mistyped the typing on L37 it should finish in `= ()` to be a class attribute.

Searching for reference usage, there is none, I think it could actually be removed. The actual check happens on the frontend and they are defined here: https://github.com/plotly/dash/blob/3488bbecacd50676cd512b1c7cdc3b4e4764acb6/dash/dash-renderer/src/actions/dependencies.js#L56-L60